src/scripts/promoCarousel.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const promoCarauselRightElemenet = document.getElementById('promoCarouselRight');
```

Comment:
literówka, jest "elemenet" zamiast element

Reply:
Może to celowe? :P 
Poprawione